fastlane/fastlane
# [action][set_github_release] support `discussion_category_name` parameter to link discussion with release

### Checklist

### Motivation and Context
- By using `discussion_category_name` parameter, we can link GitHub discussion with release
- https://docs.github.com/en/rest/releases/releases#create-a-release

### Description
- Added `discussion_category_name` parameter support to link discussion with release
- `discussion_category_name` parameter type is String and optional
- Updated unit-tests

### Testing Steps
- Use `set_github_release` action with `discussion_category_name` parameter (see unit-tests for full syntax)

### Screenshot
<img width="1000" alt="Schermafbeelding 2022-06-30 om 5 39 58 PM" src="https://user-images.githubusercontent.com/5364500/176720007-1ef212eb-f7b4-458d-8492-68887c140dbf.png">
